match/lib/match/encryption.rb

```diff
@@ -18,6 +18,9 @@ def backends
           "s3" => lambda { |params|
             params[:keychain_name] = params[:s3_bucket]
             return Encryption::OpenSSL.configure(params)
+          },
+          "gitlab_secure_files" => lambda { |params| 
+            return nil
```

Comment:
why don’t we want to encrypt with OpenSSL?

Reply:
Since Secure Files are encrypted within GitLab, I thought it wasn't necessary to encrypt again. This would also simplify the configuration since `MATCH_PASSWORD` wouldn't need to be set in this case.